node/src/types/deploy.rs

```diff
@@ -190,7 +190,7 @@ pub enum DeployConfigurationFailure {
 
     /// Invalid transfer amount.
     #[error("invalid transfer amount")]
```

Comment:
```suggestion
    /// Failed to parse transfer "amount" runtime argument.
    #[error("failed to parse transfer 'amount' runtime argument")]
```
Also please similarly update comment and `#[error]` for `MissingPaymentAmount` variant too.

Reply:
Applied suggestion